docs/Working Agreement.md

```diff
@@ -0,0 +1,166 @@
+# Working Agreement
+
+## Goal
+
+This living document represents the principles and expected behavior of everyone involved in the project. It is not
+meant to be exhaustive nor complete. The team should be accountable to these standards and revisit, review, and revise
+as needed. The agreement is signed off by everyone.
+
+## Code of Conduct
+
+We pledge to follow the [Microsoft Open Source Code of Conduct: Commercial Software Engineering Technical
+Engagements](code_of_conduct.md) which strives to ensure that every member of our team and the world-class organizations
+we engage with feels welcome and safe. In that spirit, we ask you to agree to and uphold our conduct standards as part
+of this technical engagement and to hold us to these same standards.
+
+## Communication
+
+- We communicate all information relevant to the team through the project communication Teams channel.
+- We add all research results, design documents and other technical documentation to the project repository through PRs.
+
+## Work life balance
+
+- Our office hours, when we can expect to collaborate via Microsoft Teams, phone or face-to-face are Monday to Friday
+  10AM - 3PM (PST) when possible.
+- Sometimes emails or IM messages happen when thoughts come to mind and may come outside of working hours. We are not
+  expected to answer emails or IMs past 5PM, on weekends, or when we are on holidays or vacation.
+- We work in different time zones and respect this, especially when setting up recurring meetings.
+- We record meetings when possible, so that team members who could not attend live can listen later.
+
+## Quality and not quantity
+
+- We agree on a Definition of Done for our user stories and sprints and live by it.
+- We follow engineering best practices like the [Code With Engineering
+  Playbook](https://github.com/microsoft/code-with-engineering-playbook)
+- We try to do demos every sprint when appropriate, we don't force a demo to fit just because it is the end of the
+  sprint.
+- Appropriate time to prepare demos is given and an expectation of a demo on a moments notice will be avoided.
+
+## Scrum rhythm (This will be completed after ADS)
+
+We will follow guidance from the Code With Engineering Playbook on [Agile core
+expectations](https://github.com/microsoft/code-with-engineering-playbook/blob/main/docs/agile-development/core-expectations/README.md),
+which provides guidance on the purpose and structure of the ceremonies noted below.
+
+| Activity                       | When                  | Duration    | Who          | Accountable   | Goal                                                                                                                                |
```

Comment:
Overall the timings for these seem too much. Standup + Parking lot should be 30 min. All the other meetings should be 60

Reply:
Agreed.